pyproject.toml

```diff
@@ -155,10 +155,11 @@ python = ["3.9", "3.11"]
 
 [tool.hatch.envs.docs]
 dependencies = ["tmt[docs]"]
-detached = true
 
 [tool.hatch.envs.docs.scripts]
-html = "make -C {root}/docs html"
+autodocs = "make -C {root}/docs generate-autodocs"
```

Comment:
I'm not the biggest fan of running `make` as hatch script. IMHO hatch scripts should be called from Makefile, not the other way around.  
The commands could be done in `docs` hatch environment as sphinx-api-doc fits the unwritten rule that python things go to hatch, rest to Makefile. Don't you think @happz ?

Reply:
Well, yes and no... I guess `hatch` will have to run at least some `make` targets, as `docs` is a target in `docs/Makefile`, for example, and `make` still has qualities `hatch` does not offer. But I suppose there does not have to be 1:1 mapping between hatch scripts and make targets, I can still run `make -C docs generate-autodocs` if needed.

Reply:
Looking at it again, it make sense. Thanks.  
For sake of consistency though, can we move `man` to Makefile too?

Reply:
I think we can, although I'd leave it for later patches. I think we should get rid of `html` as well, and leave just `docs`, for example. For now, I reverted most of my changes to `pyproject.toml` and `Makefile`, and I'd do the "refactoring" later.

Reply:
Seems to be outdated, closing.